api/src/main/scala/com/rasterfoundry/granary/services/ModelService.scala

```diff
@@ -15,30 +15,39 @@ import sttp.tapir.server.http4s._
 
 import java.util.UUID
 
-class ModelService[F[_]: Sync](contextBuilder: TracingContextBuilder[F], xa: Transactor[F])(
+class ModelService[F[_]: Sync](
+    contextBuilder: TracingContextBuilder[F],
+    xa: Transactor[F]
+)(
     implicit contextShift: ContextShift[F]
 ) extends GranaryService {
 
-  def createModel(model: Model.Create): F[Either[Unit, Model]] =
+  def createModel(model: Model.Create): F[Either[CrudError, Model]] =
```

Comment:
I see the `CrudError`, and I know nothing inside the endpoint can actually cause one to occur... this is a bit weird, since this is basically docs polluting application logic. I think since the auth logic moved to a separate layer it might make sense to swap this back to `Unit` (and in the other places)